tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -50,29 +50,45 @@ def run(ceph_cluster, **kw):
     objectstore_obj = objectstoreToolWorkflows(node=cephadm)
     mon_obj = MonConfigMethods(rados_obj=rados_obj)
     client_node = ceph_cluster.get_nodes(role="client")[0]
-    scrub_obj = RadosScrubber(node=cephadm)
-    obj_count = ""
+    wait_time = 45
 
     try:
         # set global autoscaler to off
         rados_obj.configure_pg_autoscaler(**{"default_mode": "off"})
         # Creating ec pool
         ec_config = config.get("ec_pool")
         pool_name = ec_config["pool_name"]
-
         req_no_of_objects = config.get("inconsistent_obj_count")
 
+        # Set the default values
+        mon_obj.remove_config(section="osd", name="osd_scrub_min_interval")
+        mon_obj.remove_config(section="osd", name="osd_scrub_max_interval")
+        mon_obj.remove_config(section="osd", name="osd_deep_scrub_interval")
+
+        if not check_for_scrub(rados_obj, wait_time):
+            log.error(
+                "The scrub operations are still in progress.Not executing the further tests"
+            )
+            return 1
+
```

Comment:
Add noscrub and deep-scrub flag before creating the pool.

Reply:
Moved the statement to before creating the pool.